laghos.cpp

```diff
@@ -1146,9 +1149,10 @@ int main(int argc, char *argv[])
                     romOptions.sampV = twparam(romOptions.window,oss+1);
                     romOptions.sampE = twparam(romOptions.window,oss+2);
 
+                    basis->LiftROMtoFOM(romS, S);
```

Comment:
It seems this can be done `if (romOptions.hyperreduce)` since the full-order ROM case should already have done the lifting.